web/thesauruses/c/17/operators.json

```diff
@@ -0,0 +1,202 @@
+    },
+    "factorial": {
+      "not-implemented": "true",
+      "name": "Factorial operator"
+    },
+    "absolute_value": {
+      "not-implemented": "true",
+      "name": "Absolute value operator"
+    },
+    "percentage": {
+      "not-implemented": "true",
+      "name": "Percentage operator"
+    },
+    "equal_to": {
+      "code": "a==b",
+      "name": "Equality operator"
+    },
+    "not_equal_to": {
+      "code": "a!=b",
+      "name": "Not equal to operator"
+    },
+    "less_than": {
+      "code": "a<b",
+      "name": "Less than operator"
+    },
+    "less_than_or_equal_to": {
+      "code": "a<=b",
+      "name": "Less than or equal to operator"
+    },
+    "greater_than": {
+      "code": "a>b",
+      "name": "Greater than operator"
+    },
+    "greater_than_or_equal_to": {
+      "code": "a>=b",
+      "name": "Greater than or equal to operator"
+    },
+    "null_coalescing": {
+      "not-implemented": "true",
+      "name": "Null coalescing operator"
+    },
+    "is": {
+      "not-implemented": "true",
+      "name": "Is operator"
+    },
+    "is_not": {
+      "not-implemented": "true",
+      "name": "Is not operator"
+    },
+    "logical_and": {
+      "code" : "if((a==1)&&(b==1))",
+      "name" : "Logical AND Operator"
+  },
+    "logical_or": {
+      "code" : "if((a==1)||(b==1))",
+      "name" : "Logical OR Operator"
+  },
+  "logical_not": {
+      "code" : "if(!a)",
+      "name" : "Logical NOT Operator"
+  },
+"bitwise_and": {
+      "code" : "a&b",
+      "name" : "Bitwise AND operator"
+  },
+ "bitwise_and_assignment" : {
+      "code" : "a &= b",
+      "name" : "Bitwise AND and assignment operator"
+  },
+"bitwise_or": {
+      "code" : "a|b",
+      "name" : "Bitwise OR operator"
+  },
+"bitwise_or_assignment": {
+      "code" : "a|=b",
+      "name" : "Bitwise OR and assignment operator"
+  },
+ "bitwise_not": {
+      "code" : "~a",
+      "name" : "Bitwise NOT operator"
+  },
+ "not_assignment": {
+        "not-implemented": "true",
+	"name" : "Bitwise NOT and assignment operator"
+ },
+"bitwise_xor": {
+      "code" : "a^b",
+      "name" : "Bitwise XOR operator"
+  },
+"bitwise_xor_assignment": {
+      "code" : "a^=b",
+      "name" : "Bitwise XOR and assignment operator"
+  },
+ "bitwise_xnor": {
+      "not-implemented": "true",
+      "name" : "Bitwise XNOR operator"
+  },
+ "bitwise_xnor_assignment": {
+      "not-implemented": "true",
+      "name" : "Bitwise XNOR and assignment operator"
+  },
+    "left_shift": {
+      "code": "a<<1",
+      "name": "Left shift bitwise operator"
+    },
+    "left_shift_assignment": {
+      "code": "a<<=1",
+      "name": "Left shift assignment operator"
+    },
+    "right_shift": {
+      "code": "a>>1",
+      "name": "Right shift bitwise operator"
+    },
+    "right_shift_assignment": {
+      "code": "a>>=1",
+      "name": "Right shift assignment operator"
+    },
+    "ternary": {
+      "code": "(a>b)? printf(\"0\") : printf(\"1\")",
```

Comment:
While this is basically right, can you simplify it? Like `expression ? true_statement() : false_statement()` or something?